### PR TITLE
Change tictok to tiktok, optimization icons

### PR DESCRIPTION
**Pull request by KarlRaphel**
## Description

Change tictok to tiktok, optimization icons in https://github.com/LawnchairLauncher/lawnicons/pull/1378/

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories, such as copyediting)

### Icons linked
* Tiktok (linked `com.ss.android.ugc.aweme/com.ss.android.ugc.aweme.splash.SplashActivity` to `@Tiktok`)

### Icons updated

* All icons in https://github.com/LawnchairLauncher/lawnicons/pull/1378/

### CODE REVIEW

**Comment by KTibow** [comment]
fyi if you have node installed, you can run `npx svgo -f svgs -o svgs` to run the real optimization
not that it matters at the end of the day where extra stuff is removed while converting to a drawable though - this pr looks fine as is

**Comment by KarlRaphel** [comment]
Thank you, I will try it next time

**Comment by KarlRaphel** [comment]
# Icons linked
* Gallery (linked `com.dot.gallery` to `@gallery`)

**Comment by KarlRaphel** [comment]
## Icons linked
* StatusLyricExt (linked `io.baolong24.statuslyricext` to `@yandex_music`)

## Icons added
* Wandoujia (`com.wandoujia.phoenix2/com.pp.assistant.activity.PPMainActivity`)

**Comment by flame-0** [comment]
Is this intended, `alook_browser.svg`?
image

**Comment by KTibow** [comment]
it's still slightly off
image
image image

File: svgs/youku.svg
Comment (x9136) [inline svgs/youku.svg:1]: Screenshot 2023-09-12 at 11 21 59

Could you connect the points on the left and increase the length on the right to correct the defects?

**Review by x9136 — CHANGES_REQUESTED** [review]
Hi, thanks for the icons, I've looked at them all and posted my comments.

I think @Chefski didn't have a chance to check everything carefully, or he decided to accept as is out of principle that an icon with flaws is better than when there is no icon.

In general, I was not very picky, but I noted the main points, including: wrong dimensions, wrong positioning and visual defects.

**Pull request closed without merging**